IBAnimatable/AnimatableTableViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  AnimatableTableViewController.swift
+//  IBAnimatable
+//
+//  Created by phimage on 26/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+open class AnimatableTableViewController: UITableViewController, ViewControllerDesignable, StatusBarDesignable, RootWindowDesignable, TransitionAnimatable, RefreshControlDesignable {
+
+  // MARK: - ViewControllerDesignable
+  @IBInspectable open var hideNavigationBar: Bool = false
+
+  // MARK: - StatusBarDesignable
+  @IBInspectable open var lightStatusBar: Bool = false
+
+  // MARK: - RootWindowDesignable
+  @IBInspectable open var rootWindowBackgroundColor: UIColor?
+
+  // MARK: - TransitionAnimatable
+  @IBInspectable  var _transitionAnimationType: String? {
```

Comment:
well seen
copy paste from `AnimatableViewController`
Maybe the variable must also be `open` ?